packages/server/dataloader/atlassianLoaders.ts

```diff
@@ -83,7 +86,9 @@ export const freshAtlassianAuth = (parent: RootDataLoader) => {
   )
 }
 
-export const jiraRemoteProject = (parent: RootDataLoader) => {
+export const jiraRemoteProject: (
+  parent: RootDataLoader
+) => DataLoader<JiraRemoteProjectKey, JiraProject | null, string> = (parent: RootDataLoader) => {
```

Comment:
This is interesting. TS can't figure out it because there's a call to parent and it requires to specify the type explicitly.  

`implicitly has type 'any' because it does not have a type annotation and is referenced directly or indirectly in its own initializer.ts(7022)`